### PR TITLE
Added posibility for admin to add promocode

### PR DESCRIPTION
**Pull request by MickhaelSh**
## Description
Added posibility for admin to add promocode to existed or new order

#### Screenshots

|         Original          |         Updated          |
| :-----------------------: | :----------------------: |
| oldPromo |newPromo |

### Checklist

- [x] 🔽 My branch is up-to-date with "development" branch
- [x] ✅ All tests passed locally
- [x] ✨ My changes working with up-to-date front-end and back-end part locally, like charm
- [x] 🔗 Link pull request to issue

### CODE REVIEW

File: src/pages/order-item/tabs/add-product-form/add-product-form.js [hunk 1]
```diff
@@ -75,95 +104,124 @@ const AddProductForm = ({
...
+          data-testid='autocomplete'
+          onInputChange={(_e, value) => {
+            setProductInput(value);
+          }}
+          noOptionsText={productAdditionalInfo.noOneProduct}
+          options={products}
+          getOptionLabel={(option) => option?.name[0]?.value}
+          onChange={(_e, value) => {
+            if (value) {
+              setSelectedProduct(value);
```
Comment (High-Voltaged) [inline src/pages/order-item/tabs/add-product-form/add-product-form.js:138]: Can be refactored to just setSelectedProduct(value || null)

File: src/pages/order-item/tabs/tests/add-product-form-tests/add-product-form.spec.js [hunk 2]
```diff
@@ -78,4 +105,30 @@ describe('tests for add-product-form component', () => {
...
+    fireEvent.change(input, { target: { value } });
+    fireEvent.click(button);
+
+    expect(await screen.findByTestId('promo-input')).toHaveValue('');
+  });
+  it('should show error after entering wrong promoCode', async () => {
+    const input = screen.getByTestId('promo-input');
+    const value = 'wall';
+    const button = screen.getByTestId('promo-button');
+    const error = 'Промокод не знайдено';
```
Comment (High-Voltaged) [inline src/pages/order-item/tabs/tests/add-product-form-tests/add-product-form.spec.js:127]: I would advise you against using hard-coded values, you could just import an error constant you created.

**Review by High-Voltaged — APPROVED** [review]

File: src/pages/order-item/tabs/tests/add-product-form-tests/add-product-form.variables.js [hunk 3]
```diff
@@ -1,3 +1,7 @@
+const {
+  getPromoCodeByCode
+} = require('../../../../promo-code/operations/promo-code.queries');
```
Comment (dmytruk-y) [inline src/pages/order-item/tabs/tests/add-product-form-tests/add-product-form.variables.js:3]: I think you should not use CommonJS import on the frontend

Comment (MickhaelSh) [reply src/pages/order-item/tabs/tests/add-product-form-tests/add-product-form.variables.js]: changed

**Review by dmytruk-y — APPROVED** [review]

**Review by High-Voltaged — APPROVED** [review]

**Pull request merged**